development.md

```diff
@@ -0,0 +1,33 @@
+# Development
+
+## What the script is:
+The [Competency script](https://github.com/sendwithus/competencies/blob/master/competencySyncScript.gs) is what makes this project work. The script is written in Google Apps Script, a language based off of JavaScript that is specific to the G Suite products. The script takes configuration values from the Google Spreadsheet to which it is attached. You are free to use the script for your own projects. Follow the instructions below to start syncing your Google Drive documents to GitHub.
```

Comment:
I think this should be just the name of the file in coding tags.

`competencySyncScript.gs`